### DISCUSSION TITLE
Limit upload size in per folder

### DISCUSSION THREAD

**Discussion by RetroChaos** [craftcms/cms#12358] [Ideas]
Hi, I'm unsure if my issue is too niche or can be better achieved via another method, but our backend users have issues with 'appropriate' file sizes, for instance uploading 2MB images for custom thumbnails or other smaller images, however there are certain folders which need to house larger PDF files for rulebooks etc.

I'm aware this issue is very behavioural.

Is there a way to overwrite Craft's max upload size in certain folders? Be it a hidden file in the folder or a config setting?

If not no worries, I will just have to FTP these larger files up since they seem very resistant to change 😞

**Comment by brandonkelly** [comment]
There’s no way to do this currently. And if we ever added it, I think it would be a volume setting, not something that could be adjusted on individual subfolders.
